testbed/test-datasets/f3/gen_file.go

```diff
@@ -0,0 +1,39 @@
+package main
```

Comment:
Quick note. There's already a scripts/random-file.sh in the repo in case you want to generate a bunch of random files, add them to the dataset directory, and use them for every test so you don't have to explicitly generate them again.

But this LGTM if you rather generate them with go instead of using the bash script :)

Reply:
Go for it!